contrib/local/CONTRACTS.md

```diff
@@ -0,0 +1,64 @@
+# Deployment and Instantiation
+
+See the list of codes that was uploaded to the testnet previously.
+```
+persistenceCore query wasm list-code --node https://rpc.testnet.persistence.one:443
+```
+
+You can set the `node` to the persistenceCore config and don't have to worry about passing that flag always
+
+```
+persistenceCore config node https://rpc.testnet.persistence.one:443
+```
+
+To upload the contract
+
+```
+RESP=$(persistenceCore tx gov submit-proposal wasm-store "path/to/the/compiled/wasm" \
```

Comment:
replace `"path/to/the/compiled/wasm"` to `"<path/to/the/compiled/wasm>"` so we know we have to replace it